tools/build_api.py

```diff
@@ -1122,19 +1122,18 @@ def get_unique_supported_toolchains(release_targets=None):
 
     return unique_supported_toolchains
 
+
+def _lowercase_release_version(release_version):
+    try:
+        return release_version.lower()
+    except AttributeError:
+        return 'all'
```

Comment:
Return 'all'?
Will this result in only printing the version warning if run with Py2?

Reply:
No?